src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double base;
+    private final double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (height * base) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle,"
+                + " area: " + calculateArea()
+                + " sq. units, height: " + height
+                + " units, base: " + base
+                + " units, color: " + getColor());
```

Comment:
Instead of concatenating strings in the `System.out.println` call, use `System.out.printf` or `String.format` for better readability and performance.